bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -416,7 +417,7 @@ public ClientConfiguration setClientReceiveBufferSize(int bufferSize) {
      * @return netty channel write buffer low water mark.
      */
     public int getClientWriteBufferLowWaterMark() {
-        return getInt(CLIENT_WRITEBUFFER_LOW_WATER_MARK, 32 * 1024);
+        return getInt(CLIENT_WRITEBUFFER_LOW_WATER_MARK, 384 * 1024);
```

Comment:
why 384?

Reply:
goes along with answer on why HWM changed.
Pretty much set to 75% of HWM after couple of quick experiments.